accern_xyme/util.py

```diff
@@ -177,7 +177,10 @@ def get_file_hash(buff: IO[bytes]) -> str:
 
 def interpret_ctype(data: IO[bytes], ctype: str) -> ByteResponse:
     if ctype == "application/json":
-        return json.load(data)
+        res = json.load(data)
+        if "errMessage" in res and res["errMessage"]:
```

Comment:
hmm, that's not good -- should be a different ctype then -- how do we know the actual output doesn't contain a "errMessage" field?

Reply:
do application/json+error